editor/src/clj/editor/app_view.clj

```diff
@@ -1,3 +1,15 @@
+;; Copyright 2020 The Defold Foundation
+;; Licensed under the Defold License version 1.0 (the "License"); you may not use
+;; this file except in compliance with the License.
+;;
+;; You may obtain a copy of the License, together with FAQs at
+;; https://www.defold.com/license
+;;
+;; Unless required by applicable law or agreed to in writing, software distributed
+;; under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+;; CONDITIONS OF ANY KIND, either express or implied. See the License for the
+;; specific language governing permissions and limitations under the License.
+
```

Comment:
Did we miss .clj files?

Reply:
We added the Defold license to all files on the `dev` branch but I would have expected this to propagate to `editor-dev` eventually?

Reply:
Hmm, yes, it's a bit confusing.